api.py

```diff
@@ -0,0 +1,37 @@
+import asyncio
+import redis
+import json
+import uuid
+import os
+
+
+class Api:
+    def __init__(self):
+        self.r = redis.StrictRedis(host='localhost', port=6379, decode_responses=True)
+
+    @asyncio.coroutine
+    def handle_api_request(self, path):
+        result = None
+
+        if path[-1] == '/':
+            path = path[:-1]
+
+        request, arg = os.path.split(path)
```

Comment:
I'd use `/api/stats` and `/api/stats?uuid=aaaa-bbbb....` and then https://docs.python.org/3/library/urllib.parse.html or aiohttp to parse the request into components.